pkgs/tools/misc/pastel-rs/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  name = "pastel-rs";
```

Comment:
```suggestion
```
The name will automatically be set by the pname and version.

Reply:
Thanks! Also, should the name just remain the same as in `pname` then at the `top-level` packages (just `pastel`)?

Reply:
looks like it should just be `pastel`. https://repology.org/project/pastel/versions